src/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <title>To Do List</title>
+    <script
+      src="https://kit.fontawesome.com/69940923dd.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <section class="main-container">
+      <div class="title">
+        <h3>Today's To Do</h3>
+        <i class="fa-solid fa-rotate fa-rotate-90"></i>
+      </div>
+      <div class="input-div">
+        <input type="text" id="input" placeholder="Add to your list.." />
+
+        <i class="fa-solid fa-arrow-turn-down fa-rotate-90"></i>
```

Comment:
- [ ] @mertkantaroglu great work implementing the changes requested, however, the arrow button is slightly outside the  container , kindly ensure its inside the container to match the design given
![image](https://user-images.githubusercontent.com/61408860/214331945-797bd983-950b-424e-9fcf-d0ec6478eb2e.png)

Reply:
Hello Daniel,
Thanks for your feedback, but in my screen it shows in the container.
![Screen Shot 2023-01-24 at 18 30 49](https://user-images.githubusercontent.com/44959066/214336729-56e9cfe3-f7aa-418a-acf7-6ffd56883680.png)

Can you please tell me how you looked at it? Then I can fix it accordingly.
Besides, is this really a big problem to postpone my approval? If you would say optional, I could fix it later on.